scanner/scanner.go

```diff
@@ -358,6 +358,8 @@ func humanTime(seconds int) string {
 // precert string as the arguments.
 //
 // This method blocks until the scan is complete.
+//
+// TODO: Scan leaks resources.
```

Comment:
can you make this `TODO(FiloSottile)` so someone coming back here later knows who to ask about this comment?